trucks.js

```diff
@@ -246,9 +252,47 @@ var foodTrucks = [
   }
 ];
 
-/* 
+var trucks = (function(){
+  var truckName = '';
+  var trucks = [];
+
+  function getTruckByDay( truck, day ){
+    if (truck.schedule.indexOf(day) !== -1){
+      return true;
+    } else {
+      return false;
+    }}
+
+  function filterTrucksByDay( day ){
+    if( day !== null ){
```

Comment:
Better to check for truthiness (`if (day) {`) or `if (!day === undefined) {` as null won't cover the more likely scenario of nothing being passed in.